tests/rptest/wasm/background_task.py

```diff
@@ -34,14 +34,14 @@ def _run(self):
     def _run_protected(self):
```

Comment:
please fix the commit message (and subject) (`rptest/wasm: background_task set _done to true within finally block`) to state what this commit fixes.